lib/app.js

```diff
@@ -42,12 +43,7 @@ class AppView extends React.Component {
   constructor(props) {
     super(props);
 
-    // Metrics
-    Metrics.launch();
```

Comment:
If possible I would  not remove the metrics, it would be better first check that we have enough data from ga and in that case, if these metrics do not give us anything else we need, we can remove them